addons/account/models/account_move.py

```diff
@@ -2554,3579 +2320,1276 @@ def _get_invoice_computed_reference(self):
         self.ensure_one()
         if self.journal_id.invoice_reference_type == 'none':
             return ''
-        else:
-            ref_function = getattr(self, '_get_invoice_reference_{}_{}'.format(self.journal_id.invoice_reference_model, self.journal_id.invoice_reference_type))
-            if ref_function:
-                return ref_function()
-            else:
-                raise UserError(_('The combination of reference model and reference type on the journal is not implemented'))
+        ref_function = getattr(self, f'_get_invoice_reference_{self.journal_id.invoice_reference_model}_{self.journal_id.invoice_reference_type}')
+        if ref_function is None:
```

Comment:
I don't think that's ever worked, without a default value as fallback `getattr` behaves the same way an equivalent attribute access does: it raises `AttributeError`.